packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -124,6 +124,12 @@ const propTypes = {
    * If the `variant` prop if set to `12-hour` for one of these supported locales, the variant will be ignored and defaults to `24-hour`.
    */
   timeVariant: PropTypes.oneOf([DateTimeUtils.FORMAT_12_HOUR, DateTimeUtils.FORMAT_24_HOUR]),
+  /**
+   * Timezone value to indicate in which timezone the date-time component is rendered.
+   * The value provided should be a valid [timezone](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) string, else will default to browser/local timezone.
+   * Note: This value is considered only for the initial render. Changes to an already set time zone will reset to the first set time zone.
+   */
+  initialTimeZone: PropTypes.string,
```

Comment:
Can you move this and `timeVariant` up so the props are in alphabetical order?

Reply:
2624eda3c88fd2434edbb3be07736b64b9ab2423 changed here